### ISSUE TITLE
False positive after using a member from an aliased import

### ISSUE THREAD

**Issue by PeterJCLaw** [TylerYep/flake8-future-annotations#13]
Given:
``` python
import blah as blah2

x: blah2.Dict[str] = 42
```
This linter will raise an error which appears spurious:
```
demo.py:1:1: FA100 Missing from __future__ import annotations but imports: blah2.Dict
```

I think the fix is that `visit_Import` should have the `asname` branch nested within the `if alias.name == "typing":` branch rather than separate from it.

**Comment by TylerYep** [comment]
Makes sense, I will accept a PR for this fix! 👍
